src/main/scala/com/lifeway/sbt/testdocker/StartTestDocker.scala

```diff
@@ -27,10 +28,14 @@ object StartTestDocker {
       } else {
         streamz.log.info("Starting testdocker image")
         Process(arg).run()
+        var attempts = 0
+        val r        = attempts = attempts + 1
+
         do {
           streamz.log.info(s"Waiting for testdocker to boot on port $hostPort")
           Thread.sleep(250)
-        } while (!isHostPortActive(hostPort))
+          attempts = attempts + 1
+        } while (!isHostPortActive(hostPort) && attempts <= 240)
```

Comment:
what does 240 represent?

Reply:
I experimented with something a long time ago and forgot I'd made that change. I thought it was on master. I'll remove it.